app/src/main/java/org/mozilla/fenix/tabstray/browser/BrowserTrayInteractor.kt

```diff
@@ -21,10 +21,21 @@ import org.mozilla.fenix.tabstray.TabsTrayStore
  */
 interface BrowserTrayInteractor : SelectionInteractor<Tab>, UserInteractionHandler {
 
+    /**
+     * Open a tab.
+     *
+     * @param tab [Tab] to open in browser.
+     * @param source app feature from which the [tab] was opened.
+     */
+    fun open(tab: Tab, source: String? = null)
```

Comment:
Found it a bit strange that the contract for browser tray says that an item can be closed but cannot be opened.
Added a open method here perfectly suited for just the tabs tray needs - knowing which tab to open and from which tabs try sub-feature.